### PR TITLE
feat: search upward for a .czrc file

### LINKED ISSUE #47: .czrc is not used at root directory if package.json does not exist
Hello,

I noticed that if there is not `package.json` file at root directory then the `.czrc` file is not used.

It looks to be something supported by `cz-cli` though.

Let me know how I could help.
Cheers.

### PR DESCRIPTION
**Pull request by ngryman**
Fixes #47 
Search upward for a `.czrc` file. Published as `1.2.3-1` with the `next` tag.

@B-Galati Could you try this version and let me know if it resolves your issue?

### CODE REVIEW

**Comment by B-Galati** [comment]
Works like a charm thank you :pray:

File: index.js [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
+async function getConfig() {
+  const defaultConfig = {
+    types,
+    symbol: false,
+    skipQuestions: [''],
+    subjectMaxLength: 75,
+    conventional: false
+  }
+
+  const config = (await loadConfigFrom('package.json')) || (await loadConfigFrom('.czrc'))
```
Comment (B-Galati) [inline index.js:30]: I feel like it's still necessary to look explicitly in the home directory like `readFromGlobalCzrc` was doing, no?

Comment (ngryman) [reply index.js:30]: From my experience, most projects sit under the home directory, at least on *NIX systems. So the current logic should capture the home directory in most cases. However, I don't really know what's the tendency on Windows systems 🤔 

In any case, it is pretty simple to implement so I'll just add it. Thanks for the raising it!

Comment (B-Galati) [reply index.js]: From my experience it's something very personal that cannot be assumed. Thanks for fixing it so quickly! Really appreciate it!

**Pull request merged**